frontend/src/component/changeRequest/ChangeRequestOverview/ReviewButton/ReviewButton.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { useRequiredPathParam } from 'hooks/useRequiredPathParam';
+import { useChangeRequest } from 'hooks/api/getters/useChangeRequest/useChangeRequest';
+import { useChangeRequestApi } from 'hooks/api/actions/useChangeRequestApi/useChangeRequestApi';
+import { formatUnknownError } from 'utils/formatUnknownError';
+import useToast from 'hooks/useToast';
+
+import {
+    Button,
+    Grow,
+    Paper,
+    Popper,
+    MenuItem,
+    MenuList,
+    ClickAwayListener,
+} from '@mui/material';
+
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+
+const options = ['Approve change', 'Reject change'];
+
+export const ReviewButton = () => {
+    const projectId = useRequiredPathParam('projectId');
+    const id = useRequiredPathParam('id');
+    const { refetchChangeRequest } = useChangeRequest(projectId, id);
+    const { setToastApiError, setToastData } = useToast();
+
+    const { changeState } = useChangeRequestApi();
+
+    const [open, setOpen] = React.useState(false);
+    const anchorRef = React.useRef<HTMLButtonElement>(null);
+
+    const onMenuItemClick = async (index: number) => {
```

Comment:
I'd split this into separate fns

Reply:
Addressed here: https://github.com/Unleash/unleash/pull/2322/commits/2f766a26df55d64bc600020aa3e5f9020f10abdb